setup.cfg

```diff
@@ -10,8 +10,9 @@ dev =
 doc =
     matplotlib >= 1.5.3
     numpydoc
-    sphinx >=0.3,<3
+    sphinx ~= 5.3
     texext
+    toml
```

Comment:
Just a note that we should be using `tomli; python_version < "3.11"` and importing as:

```Python
try:
    import tomllib
except ImportError:
    import tomli as tomllib
```

The toml package is unmaintained, and `tomli` has become `tomllib` in the stdlib.

Reply:
My bad, resolved in 4b693b84fd267f4a53a65070487c2d90d8f64ad6.